### PR TITLE
Fixes camera MDP term name and reprojection docstrings

### PR DESCRIPTION
**Pull request by Mayankm96**
# Description

For all observation-related terms, we follow the noun name for the function. However, the camera one was called `grab_images`. This MR renames it to `image` to make it consistent with the others.

Also the docstring for the math-utils were messed up. According to Google's docstyle, the first line should be a short summary and a detailed summary should move to the next paragraph. This MR fixes that as well.

Lastly, the name of the function `convert_perspective_depth_to_orthoginal_depth` has too much redundancy. The MR renames it to `orthogonalize_perpsective_depth`.

All these are breaking changes with no backwards compatibility (since they were added post-last release anyway).

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

## Checklist

- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `./isaaclab.sh --format`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have updated the changelog and the corresponding version in the extension's `config/extension.toml` file
- [x] I have added my name to the `CONTRIBUTORS.md` or my name already exists there

### CODE REVIEW

File: source/extensions/omni.isaac.lab/omni/isaac/lab/utils/math.py [hunk 1]
```diff
@@ -988,12 +988,7 @@ def transform_points(
 
 @torch.jit.script
 def unproject_depth(depth: torch.Tensor, intrinsics: torch.Tensor) -> torch.Tensor:
```
Comment (glvov-bdai) [inline source/extensions/omni.isaac.lab/omni/isaac/lab/utils/math.py]: Although it mentions in the doc string that depth here should be orthogonal maybe good to also mention it in the method name like `unproject_orthogonal_depth` as I feel like the doc string comment could be missed

**Review by glvov-bdai — APPROVED** [review]
lgtm

**Review by jtigue-bdai — APPROVED** [review]

File: source/extensions/omni.isaac.lab/config/extension.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
 
 # Note: Semantic Versioning is used: https://semver.org/
-version = "0.24.13"
+version = "0.24.15"
```
Comment (Mayankm96) [inline source/extensions/omni.isaac.lab/config/extension.toml]:
```suggestion
version = "0.24.16"
```

File: source/extensions/omni.isaac.lab/omni/isaac/lab/utils/math.py (hunk 1 above)
Comment (Mayankm96) [reply source/extensions/omni.isaac.lab/omni/isaac/lab/utils/math.py]: I added a parameter to the function which user can set as True/False for orthogonal/perspective depth image respectively. This is probably less breaking.

**Pull request merged**